docker-compose.yaml

```diff
@@ -52,6 +52,7 @@ services:
       - SANDBOX_PROTOCOL=${SANDBOX_PROTOCOL:-}
       - START_WRITERS=${START_WRITERS:-}
       - CONTRACT_ID=${CONTRACT_ID:-}
+      - CHAIN_CACHE_TYPE=${CHAIN_CACHE_TYPE:-}
```

Comment:
We also need to add the LMDB path to the list of volume mount above.

Reply:
added here! [46b3ae8](https://github.com/ar-io/ar-io-node/pull/60/commits/46b3ae82626ec2f64095b70cc528245942e6ee6e)